bcda/cli.go

```diff
@@ -201,17 +293,64 @@ func sortCCLFFiles(cclf0, cclf8, cclf9 *[]cclfFileMetadata, skipped *int) filepa
 			*skipped = *skipped + 1
 			return nil
 		}
-		if metadata.cclfNum == 0 {
-			*cclf0 = append(*cclf0, metadata)
-		} else if metadata.cclfNum == 8 {
-			*cclf8 = append(*cclf8, metadata)
-		} else if metadata.cclfNum == 9 {
-			*cclf9 = append(*cclf9, metadata)
+
+		// if we get multiple sets of files relating to the same aco for attribution purposes, concat the year
+		key := fmt.Sprintf(metadata.acoID+"_%d", metadata.timestamp.Year())
```

Comment:
I'm not clear on what this is handling-- is it for the claims run-out period?

Reply:
@knollfear mentioned that we could potentially have multiple sets of files relating to attribution. Based on this being true, a regular ole' (key: acoID) here wouldn't work but can change if this isn't a real case.

Reply:
Ah, got it. The run-out period is when that could occur. In that case, we'd want to use the performance year from the filename, indicated by the `Y**` part. Two files from the same month would both have the current year in the timestamp, vs. the performance year being different (e.g., one `Y18` and one `Y19`). 

The CCLF8 branch parses performance year in `getCCLFFileMetadata()`, so maybe it makes sense to copy that function over to this branch?